docs/source/release/v3.14.0/ui.rst

```diff
@@ -101,3 +101,11 @@ Changes
  - The miniplot on the pick tab of the instrument view now shows the HKL values for peaks when viewing a summed collection of detectors.
 
 :ref:`Release 3.14.0 <v3.14.0>`
+
+Logging
+-------
+
+changes
+#######
+
+ - Increased the debug level of messages when creating an instrument geometry.
```

Comment:
I would write something like:
Increased the log level from information to notice when creating an instrument geometry